clarify sso okta doc (#8708)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/manage-access/scim-okta.md

## SCIM username format

> For <Constant name="dbt_platform" /> SCIM with Okta, `userName` **must be the user's email address**. <Constant name="dbt_platform" /> uses `userName` to look up existing users during SCIM sync. If Okta sends a non-email value (such as an Okta internal ID like `00u...` or an employee ID), <Constant name="dbt_platform" /> cannot match the existing user and provisioning will fail.

Reviewer: Saying "it must be the users email address" may add a level of questions (what if the user has email aliases) and might be better to stick with the language that it must be in the email address format
```suggestion
For <Constant name="dbt_platform" /> SCIM with Okta, `userName` **must be the email address format**. <Constant name="dbt_platform" /> uses `userName` to look up existing users during SCIM sync. If Okta sends another format (such as an Okta internal ID like `00u...` or an employee ID), <Constant name="dbt_platform" /> cannot match the existing user, and provisioning will fail.
```

If your Okta configurations map the `Username` field to a different attribute, set your Okta app config to `Email`:

1. Open the SAML app created for the dbt integration.
2. In the **Sign on** tab, click **Edit** in the **Settings** pane.
3. Set the **Application username format** field to **Email**.
4. Click **Save**.

#### What you'll see if misconfigured

> If `userName` is not an email address, you may encounter errors like:

Reviewer: To be consistent with the prior section
```suggestion
If `userName` is not configured for `Email`, you may encounter errors like:
```

- Okta reports **"User already exists"** when attempting to provision a user.

> - SCIM logs show a filter like `userName eq "00uXXXXXXXXX"` (an Okta internal ID instead of an email).

Reviewer 2: I'm not sure if you can see these logs in Okta, this might be more of a DD thing. @alexlod any chance you can check your Okta logs and see what it displays for this error?

Reviewer 3: Unfortunately Okta logs don't show anything. The error is displayed in the Assignment tab. There's a red exclamation mark next to the problematic user, and the error message is somewhat generic about a "conflict." Unfortunately I don't have the exact error available anymore but can get it if needed.

> - <Constant name="dbt_platform" /> returns no match on the SCIM `GET` request, then the subsequent `POST` to create the user fails with a conflict because the user already exists under their email.

Reviewer 2: This is more internal, I don't know if Okta gives that much detail

Reviewer 3: Agree with Allie. Unfortunately the Okta logs are not very helpful :(

Author: might be good to just remove this since it's not helpful
